content/2_Bronze/Intro_DS.mdx

```diff
@@ -710,6 +709,261 @@ See [here](/general/notes-cpp-ds).
 
 </LanguageSection>
 
+## Quiz
+
+<LanguageSection>
+<CPPSection>
+<Quiz>
+  <Quiz.Question>
+    How do you count the number of items in an `std::vector`? Suppose we named the vector `v`.
+    <Quiz.Answer correct>
+      `v.size()`
+      <Quiz.Explanation>
+        Correct. `size()` returns the number of elements in the data structure!
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      `v.length()`
+      <Quiz.Explanation>
+        Incorrect. The method `length()` does not exist on an `std::vector`
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      `v.count()` 
+      <Quiz.Explanation>
+        Incorrect. The method `count()` does not exist on an `std::vector`
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      `v.resize()`
+      <Quiz.Explanation>
+        Incorrect. The method `resize()` resizes the vector to the specified number of elements
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    Which of the following would print out `5`?
+    <Quiz.Answer correct>
+      ```cpp
+      int a = 3, b = 4, c = 5;
+      tuple<int, int, int> t = tie(a, b, c);
+      cout << get<2>(t);
+	  ```
+      <Quiz.Explanation>
+        Correct. `get<i>(t)`: Returns the $i$-th element of the tuple `t`
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      ```cpp
+      int a = 3, b = 4, c = 5;
+      tuple<int, int, int> t = tie(a, b, c);
+      cout << t.get(2);
+	  ```
+      <Quiz.Explanation>
+        Incorrect. `t.get()` is not a valid method.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
```

Comment:
seems odd to have a question with multiple correct answers when you can only select one

Reply:
Or add something like "If there are multiple correct answers, any of them will be accepted."